lib/__tests__/collective-sections.test.js

```diff
@@ -16,10 +22,8 @@ describe('Collective sections', () => {
           'updates',
           'conversations',
           'budget',
-          'events',
```

Comment:
No more events for collectives by default?

Reply:
Not until the feature flag is removed - test would break without it